tests/gsrpc/extrinsics/dataSubmitAndWatch/dataSubmitAndWatch.go

```diff
@@ -45,10 +46,13 @@ func main() {
 
 	// Set data and appID according to need
 	size := 0
-	if config.Size == 0  || config.Size > -1 {
+	if config.Size <= 0 {
 		size = 100
+	} else {
+		size = config.Size
 	}
```

Comment:
Maybe:
```
size := 100
if config.Size > 0 {
    size = config.Size
}
```

Reply:
well suppose the config.size is missing just wanted to take a default size of 100.